btr-web/btr-main-app/utils/significant-individual.ts

```diff
@@ -0,0 +1,51 @@
+/** Get a new significant individual with all default fields
+ * @param startDate expected iso date string (YYYY-MM-DD)
+*/
+export function getNewDefaultSI (startDate: string) {
```

Comment:
Nitpick I guess, btu I would call this empty instead of default. But all good for this review.

Reply:
right on I agree with you I'll update